pkg/statistics/handle/usage/predicate_column_test.go

```diff
@@ -175,7 +175,7 @@ func TestAnalyzeTableWithTiDBPersistAnalyzeOptionsDisabled(t *testing.T) {
 	// Analyze again, it should use the predicate columns.
 	tk.MustExec("analyze table t")
 	tk.MustQuery("select table_name, job_info from mysql.analyze_jobs order by id desc limit 1").Check(
-		testkit.Rows("t analyze table columns a with 256 buckets, 100 topn, 1 samplerate"),
+		testkit.Rows("t analyze table column a with 256 buckets, 100 topn, 1 samplerate"),
```

Comment:
Why not columns in here ? It seems that t table has two columns

Reply:
Because we only analyze predicate columns here. We only used column `a` above.